bool agvAtSmf
bool agvPrbtChangingBox
bool agvPrbtChangingPen
bool agvPrbtAtHome